plugins/scaffolder-backend-module-confluence-to-markdowwn/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "@backstage/plugin-scaffolder-backend-module-confluence-to-markdowwn",
```

```suggestion
  "name": "@backstage/plugin-scaffolder-backend-module-confluence-to-markdown",
```
One `w` too much here right?